docs/manual/docs/cmd/graph/schemaextension/schemaextension-set.md

```diff
@@ -0,0 +1,60 @@
+# graph schemaextension add
+
+Updates a Microsoft Graph schema extension
+
+## Usage
+
+```sh
+graph schemaextension set [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`-i, --id <id>`|The unique identifier for the schema extension definition
+`-d, --description [description]`|Description of the schema extension
+`-s, --status [status]`|The lifecycle state of the schema extension. Accepted values are 'Available' or 'Depecrated'
+`-t, --targetTypes <targetTypes>`|Comma-separated list of Microsoft Graph resource types the schema extension targets
+`-p, --properties`|The collection of property names and types that make up the schema extension definition formatted as a JSON string
```

Comment:
Missing designation of the property: it is required `<properties>` or optional `[properties]`?